lib/react_client.dart

```diff
@@ -40,49 +40,35 @@ newJsMap(Map map) {
   return JsMap;
 }
 
-/**
- * Type of [children] must be child or list of childs, when child is JsObject or String
- */
+/// Type of [children] must be child or list of children, when child is [JsObject] or [String]
 typedef JsObject ReactComponentFactory(Map props, [dynamic children]);
 typedef Component ComponentFactory();
 
-/**
- * The type of a ref specified as a callback.
- *
- * See <https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute>.
- */
+/// The type of [Component.ref] specified as a callback.
+///
+/// See: <https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute>
 typedef _CallbackRef(componentOrDomNode);
 
-/**
- * A Dart function that creates React JS component instances.
- */
+/// Creates ReactJS [Component] instances.
 abstract class ReactComponentFactoryProxy implements Function {
-  /**
-   * The type of component created by this factory.
-   */
+  /// The type of [Component] created by this factory.
```

Comment:
This reference to `[Component]` isn't valid; this could be a DOM component, which has nothing to do with the `Component` class.

Should just be "component", not "[Component]".

Same goes for the rest the comments in this class.